app/helpers/template_helpers.py

```diff
@@ -54,14 +54,20 @@ def __init__(
 
     @property
     def context(self) -> dict[str, Any]:
+        options: dict = {}
+        if self._survey_type:
+            options |= {
+                "cookie_settings_url": self._survey_config.cookie_settings_url,
+                "cookie_domain": self._survey_config.cookie_domain,
+            }
```

Comment:
I would simplify this. At the moment, this is creating an empty dict, conditionally adding to it, then creating another dict which this unpacks this dict into. 

It should only need 1 dict. I would do something like this to simplify it.

```py
        context = {
            "sign_out_button_text": self._survey_config.sign_out_button_text,
            "account_service_my_account_url": self._survey_config.account_service_my_account_url,
            "account_service_log_out_url": self._survey_config.account_service_log_out_url,
            "account_service_todo_url": self._survey_config.account_service_todo_url,
            "contact_us_url": self._survey_config.contact_us_url,
            "thank_you_url": url_for("post_submission.get_thank_you"),
            "page_header": self.page_header_context,
            "service_links": self.service_links_context,
            "footer": self.footer_context,
            "languages": get_languages_context(self._language),
            "theme": self._survey_config.design_system_theme,
            "language_code": self._language,
            "survey_title": self._survey_title,
            "cdn_url": self._cdn_url,
            "address_lookup_api_url": self._address_lookup_api,
            "data_layer": self.data_layer_context,
            "include_csrf_token": self._include_csrf_token,
            "google_tag_manager_id": self._google_tag_manager_id,
            "google_tag_manager_auth": self._google_tag_manager_auth,
            "survey_type": self._survey_type,
        }
        if self._survey_type:
            context["cookie_settings_url"] = self._survey_config.cookie_settings_url
            context["cookie_domain"] = self._survey_config.cookie_domain

        return context
```

Reply:
Looks better!